remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/models/JsonRemoteMessagingConfig.kt

```diff
@@ -26,7 +26,8 @@ data class JsonRemoteMessage(
     val id: String,
     val content: JsonContent?,
     val exclusionRules: List<Int>?,
-    val matchingRules: List<Int>?
+    val matchingRules: List<Int>?,
+    val translations: Map<String, JsonContentTranslations>?
```

Comment:
Same comment as in previous PRs. As it is a json model and we're using list and map, `null` should not be necessary

Reply:
done!